src/EditorFeatures/Core/Implementation/DocumentationComments/AbstractDocumentationCommentCommandHandler.cs

```diff
@@ -334,10 +334,18 @@ private void AddLineBreaks(SourceText text, IList<string> lines)
             CancellationToken cancellationToken)
         {
             // Find the documentation comment before the new line that was just pressed
-            var token = GetTokenToRight(syntaxTree, originalPosition, cancellationToken);
-            if (!IsDocCommentNewLine(token) || token.SpanStart != originalPosition)
+            var token = GetTokenToLeft(syntaxTree, position, cancellationToken);
+            if (!IsDocCommentNewLine(token))
             {
-                return false;
+                // See PressingEnter_InsertSlashes11 for an example of
+                // a case where multiple skipped tokens trivia appear at the same position.
+                // In that case, we need to ask for the token from the next position over.
+                token = GetTokenToLeft(syntaxTree, position + 1, cancellationToken);
```

Comment:
i think you would only want to do this if the token you got back had ceratin characteristics (like it has trailing skipped tokens).